extensions/stackdriver/metric/record.cc

```diff
@@ -66,7 +66,7 @@ void record(bool is_outbound, const ::wasm::common::NodeInfo &local_node_info,
        {requestProtocolKey(), request_info.request_protocol},
        {serviceAuthenticationPolicyKey(),
         request_info.mTLS ? kMutualTLS : kNone},
-       {destinationServiceNameKey(), request_info.destination_service_host},
+       {destinationServiceNameKey(), request_info.destination_service_name},
```

Comment:
If the request was sent to an IP, we will still parse the 1st element of it.
Can you verify? No need to block this PR though

Reply:
The 1st element will be parsed when the cluster name follows istio convention. We can add a test for ingressgateway stats in istio/istio repo where it uses an IP.

Reply:
Are you asking whether we should parsing "10.0.0.123", for example? It would be weird to have destination_service_name "10".

Reply:
I think Mandar does not want that and current logic would not cause an IP to be parsed. We can verify it in istio/istio test.